src/popup/pages/AppsBrowser.vue

```diff
@@ -164,8 +174,14 @@ export default defineComponent({
         shareWalletInfoInterval = setInterval(
           () => {
             try {
-              if (sdk._clients.get(currentClientId.value)) {
+              const rpcClient = sdk._clients.get(currentClientId.value);
+              if (rpcClient) {
                 sdk.shareWalletInfo(currentClientId.value);
```

Comment:
Isn't it a promise?
```suggestion
                await sdk.shareWalletInfo(currentClientId.value);
```

Reply:
I don't agree with your suggestion @subhod-i because I don't see any reason why the rest of the code should wait for the line to be resolved.